### PR TITLE
py-pkginfo: add 1.9.6

### PR DESCRIPTION
**Pull request by manuelakuhn**
https://pypi.org/project/pkginfo/#files

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-pkginfo/package.py [hunk 1]
```diff
@@ -12,9 +12,9 @@ class PyPkginfo(PythonPackage):
     homepage = "https://code.launchpad.net/~tseaver/pkginfo/trunk"
     pypi = "pkginfo/pkginfo-1.5.0.1.tar.gz"
 
+    version("1.9.6", sha256="8fd5896e8718a4372f0ea9cc9d96f6417c9b986e23a4d116dda26b62cc29d046")
     version("1.8.3", sha256="a84da4318dd86f870a9447a8c98340aa06216bfc6f2b7bdc4b8766984ae1867c")
     version("1.7.1", sha256="e7432f81d08adec7297633191bbf0bd47faf13cd8724c3a13250e51d542635bd")
     version("1.5.0.1", sha256="7424f2c8511c186cd5424bbf31045b77435b37a8d604990b79d4e70d741148bb")
 
     depends_on("py-setuptools", type=("build", "run"))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-pkginfo/package.py]: Is setuptools used at run-time?

Comment (manuelakuhn) [reply var/spack/repos/builtin/packages/py-pkginfo/package.py]: In pkginfo/tests I see `import pkg_resources`, That's why I left it.

Comment (adamjstewart) [reply var/spack/repos/builtin/packages/py-pkginfo/package.py]: Wasn't sure if that was only for unit tests or not

Comment (manuelakuhn) [reply var/spack/repos/builtin/packages/py-pkginfo/package.py]: I was not sure either. I just tested and for me all version install with `--test=root` also if `py-setuptools` is only a build dependency. Should I remove it then?

Comment (adamjstewart) [reply var/spack/repos/builtin/packages/py-pkginfo/package.py]: Yeah let's keep it build-only

Comment (manuelakuhn) [reply var/spack/repos/builtin/packages/py-pkginfo/package.py]: It's removed now :-)

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**